control-plane-operator/controllers/hostedcontrolplane/cloud/ccm/platform/kubevirt/kubevirt.go

```diff
@@ -0,0 +1,124 @@
+package kubevirt
+
+import (
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/manifests"
+	"github.com/openshift/hypershift/support/util"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/pointer"
+)
+
+var (
+	// TODO nargaman need to add to images
+	image = "quay.io/nargaman/kubevirt-cloud-controller-manager:v0.1.0"
```

Comment:
This is temporary image
Working on adding this image to the ReleaseImage, will change it when the image would be added